### PR TITLE
Mm102 homepage background image

### PR DESCRIPTION
**Pull request by claudiaGiancola**
## Description of changes - what did you do
Created new .tsx component "DisplayBgImage" to get the “picture of the day” and display it as background image of the Home.tsx component.

It's done applying "display: grid" on the main element of the Home body, so that then the content (content has to be put inside the <div class="home-content block">) appears as overlayed above the background image (main is the parent, content and background img are its children).

## Things to Check
- [ ok ] Have all the requirements of your ticket have been met
- [ n/a ] Have you added tests if needed for your ticket
- [ n/a ] Are all the tests passing

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,11 +15,11 @@ function App() {
         <Header/>
         <Routes>
           <Route path='/'
-          element={<Home/>}/>
+            element={<Home />} />
```
Comment (Kiran-R-K) [inline src/App.tsx:18]: good job fixing the indentation - thanks for catching it

File: src/Home/Home.tsx [hunk 2]
```diff
@@ -1,27 +1,35 @@
...
-    const [submitStatus, setSubmitStatus] = useState(false);  
+    const [username, setUsername] = useState("");
+    const [submitStatus, setSubmitStatus] = useState(false);
 
     const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
         setSubmitStatus(true);
     }
 
+    const homeBgImg = DisplayBgImage();
```
Comment (Kiran-R-K) [inline src/Home/Home.tsx]: when we name things, we want to make it as clear as possible what that thing is/is doing, so it's best to leave out abbreviations and write out full words instead.
I would suggest renaming this to `homeBackgroundImage`

File: src/images/DisplayBgImage.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+interface PictureOfDay {
+    date: string,
+    explanation: string,
+    hdurl: string,
+    title: string,
+    url: string
+}
+
+const DisplayBgImage = () => {
```
Comment (Kiran-R-K) [inline src/images/DisplayBgImage.tsx]: Similar to the comment above please can this be renamed as `DisplayBackgroundImage` (please also rename the filename to match)

**Review by Kiran-R-K — CHANGES_REQUESTED** [review]
It's looking really good - I have a couple of renaming requests, let me know when you've done them and I can re-review

File: src/App.tsx (hunk 1 above)
Comment (claudiaGiancola) [reply src/App.tsx:18]: The credit goes to the marvellous automatic "Format Selection" of VS code

**Review by Kiran-R-K — APPROVED** [review]
really good job

**Pull request merged**